### DISCUSSION TITLE
Removal of personal images

### DISCUSSION THREAD

**Discussion by Senor-Ducky** [Noob-Coders-Gang/Chintu-Bot-Web#4] [General]
please remove my image from the about section, you are free to keep yours, if it is not removed by today i will be shutting down the site as well as the bot.

**Comment by ProgrammerGaurav** [comment]
![image](https://user-images.githubusercontent.com/37450514/112268044-5db71b00-8c9c-11eb-8642-af0e5b439489.png)
